test_configs/lat_bw.json

```diff
@@ -6,16 +6,13 @@
 	],
 	"class_function": [
 		"FT_FUNC_SEND",
-		"FT_FUNC_SENDV",
-		"FT_FUNC_SENDMSG"
```

Comment:
To clarify, is the idea to keep send variants to provider specific test configs?